mojave/code/modules/mob/living/carbon/human/human_armor.dm

```diff
@@ -0,0 +1,91 @@
+/mob/living/carbon/human/getsubarmor(def_zone, d_type)
+	if(!def_zone)
+		//no averaging values when no bodypart is specified, that's stupid
+		return 0
+	if(isbodypart(def_zone))
+		return checksubarmor(def_zone, d_type)
+	var/obj/item/bodypart/affecting = get_bodypart(check_zone(def_zone))
+	if(affecting)
+		//If a specific bodypart is targeted, check how that bodypart is protected and return the value.
+		return checksubarmor(affecting, d_type)
+
+//We only get the most superficial edge protection, no stacking!
+/mob/living/carbon/human/get_edge_protection(def_zone)
+	var/obj/item/bodypart/affecting
+	if(def_zone)
+		if(isbodypart(def_zone))
+			affecting = def_zone
+		else
+			affecting = get_bodypart(check_zone(def_zone))
+
+	//No support for averaging out edge protection because it'd be fucking stupid
+	if(!affecting)
+		return 0
+
+	var/list/clothings = clothingonpart(affecting)
+	for(var/obj/item/clothing in clothings)
```

Comment:
Is this supposed to return the highest edge protection?  From what it reads, it will just return the rating of the first piece of clothing on a bodypart, which could be the suit, undersuit, or something else.  Even if let's say the undersuit had no edge protection, if it's first in the list would the suit edge protection not apply?

Reply:
Will now return the highest found.